dmoj/executors/BF.py

```diff
@@ -26,9 +26,10 @@ class Executor(CExecutor):
     test_program = ',+[-.,+]'
 
     def __init__(self, problem_id, source_code, **kwargs):
+        source_code = source_code.decode('utf-8')
         if source_code.count('[') != source_code.count(']'):
             raise CompileError('Unmatched brackets\n')
-        code = template % (''.join(itertools.imap(trans.get, source_code, itertools.repeat(''))))
+        code = (template % (''.join(map(trans.get, source_code, itertools.repeat(''))))).encode('utf-8')
```

Comment:
six.moves.map, or you are going to slow this down.

Reply:
Done